eth2/types/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "types"
```

Comment:
`types` is already an [existing crate](https://crates.io/crates/types). If we ever want to publish to crates.io, this might be an issue.

Reply:
Fair enough, thanks for the heads up!  I think we can cross this bridge if/when we come to it -- seems fairly easy to change this later if we want to publish this as a crate (though I doubt we'd want to do that any time soon if ever, since the types in the simulation may in some ways be specific to the needs of the simulation, but I could be wrong).